Beyond06-SOW.md

```diff
@@ -0,0 +1,346 @@
+
+The purpose of this demo is to show your development process and should includes:
+
+1. Project background - Explain in general the motivation and the requirements.
+2. Architecture -adding a chart that describes all system components.
+3. Results (features)
+4. Demo - video that presents your full system (UI, DB, APIs, CI)
+5. Development processing - share the process you made
+6. Development challenges
+7. What did you learn
+
+## Division into project groups (till Oct22th)
+
+After the first 2 weeks you'll be divided into 4 groups.
+
+Each group will have mentors:
+
+* group1: Yariv & Arie & Shimrit & Omer
+* group2: Kobi & Michal & Nati & Nahshon
+* group3: Barak & Daniel & Guy & Eran
+* group4(AWS): Yuval & Danielle B. & Eddy & Sarah & Or
+
+## Project ideas
+
+* Platform for freelancers to offer their services
+* P2P Lending service
+* Dating website
+* Drop shipping web store - selling other people's products on your website
+* Platform for finding part time jobs for students
+
+## Syllabus (list of sessions)
+
+| Session number | Date                   | Topic(s)                                                                                                                                                                                                                                                     | Lecturer             |
+|----------------|------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------|
+| -              | 10.10                   | Self learning: Tool introduction challenge                                                                                                                                                                                                                   | (Self work)          |
+| 1              | 17.10                  | Intro session (housekeeping) ~50 min +Talk about initial challenges and student selection                                                                                                                                                                    | Liora Milbaum        |
+|                |                        | Modern software applications - Architecture, Tooling and development processes.                                                                                                                                                                              | Barak Korren         |
+|                |                        | Incorporate presenting of the 1st exercise and the skill evaluation form in the 1st lecture                                                                                                                                                                  | Barak Korren         |
```

Comment:
The above line is probably not something the student should see - it looks like a task I wrote for myself...